chopper/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 4.0.0-nullsafety.0
+
+- Null safety
+- Support OPTIONS request
+- Support passing body to GET request
+
```

Comment:
```suggestion
- Sound null safety support
- Support for OPTIONS requests
- Support for passing data in the body of GET requests (antipattern, but requested)
- Support for OkHttp-like Authenticator implementations
- Updated public API documentation and how-tos

```

Reply:
I believe we can drop the [3.0.7 milestone](https://github.com/lejard-h/chopper/milestone/1) altogether and release the past couple of months' updates with 4.0.0-nullsafety.0. What do you think @JEuler @lejard-h ? 🙂

Reply:
Also, it would be nice to mention people who made this release possible (including ourselves in some way of course) - who contributed to chopper.

Maybe in a form of a short list.

Reply:
Of course, I think we can. Also, we can set the version to 4.0.0, I think. :) 

I think the ist of contributed guys is a cool idea. I saw the contribution list on https://github.com/mobxjs/mobx.dart, a table.

Reply:
I will add the contribution list :)
However I will keep the 4.0.0-nullsafety.0 and wait for feedback (if any) since it's really the first release and I already saw some improvment.

Reply:
I'm with @lejard-h on this one, this is a big "bad" breaking update. 🙂  Let's go slow on the public version number (even if we think that everything's fine 😄 ).

Reply:
Another topic (I just didn't find a better place to ask this where you can respond), will this release update the gitbook pages, or does that have to be done separately?

Reply:
yes sorry, forgot to answer that, It will update the documentation yes

But maybe I can configure gitbook to track change on `develop` branch instead of `master`

Reply:
I don't think tracking `develop` with gitbook is a good idea. The documentation should be aligned with the publicly released stuff. If we have a well-set-up CI (and I believe we do now 🙏 ), releasing a new minor version even if just the docs change wouldn't be too big of a problem.

Wdyt?

Reply:
@lejard-h Is the suggestion for the changelog OK at the beginning of this thread?